source/api/image/3.0/index.md

```diff
@@ -477,92 +477,126 @@ Servers should also support [CORS][cors-response] on Image Information responses
 {: .api-table}
 
+``` json-doc
+{
+  "@context": "http://iiif.io/api/image/{{ page.major }}/context.json",
+  "id": "http://www.example.org/image-service/abcd1234/1E34750D-38DB-4825-A38A-B60A345E591C",
+  "type": "ImageService3",
+  "protocol": "http://iiif.io/api/image",
+  "width": 6000,
+  "height": 4000,
+  "sizes": [
+    {"width": 150, "height": 100},
+    {"width": 600, "height": 400},
+    {"width": 3000, "height": 2000}
+  ]
+}
+```
+
+### 5.4. Tiles
+
+The JSON response has the `tiles` property while describes a set of image regions that have a consistent height and width, over a series of resolutions, that can be stitched together visually.
+
+| Property   | Required? | Description |
+| ---------- | --------- | ----------- |
+| `tiles` | Optional | A list of descriptions of the parameters to use to request regions of the image (tiles) that are efficient for the server to deliver. Each description gives a width, optionally a height for non-square tiles, and a set of scale factors at which tiles of those dimensions are available. |
+{: .api-table}
+
 The objects in the `tiles` list have the properties in the following table. The `width` and `height` should be used to fill the region parameter and the `scaleFactors` to complete the size parameter of the image URL. This is described in detail in the [Implementation Notes][implementation-notes].
 
-The `width` of a tile, or the combination of `width` and `height` if `height` is specified, _MUST_ be unique among the members of the `tiles` list.
 
-| Tile Object Property | Required? | Description |
+| Property | Required? | Description |
 | ---------- | -------- | ----------- |
 | `type`    | Optional | The type of the Tile. If present, the value _MUST_ be the string `Tile`. |
 | `scaleFactors` | Required | The set of resolution scaling factors for the image's predefined tiles, expressed as positive integers by which to divide the full size of the image. For example, a scale factor of 4 indicates that the service can efficiently deliver images at 1/4 or 25% of the height and width of the full image. A particular scale factor value _SHOULD_ appear only once in the `tiles` list. |
 | `width` | Required | The width in pixels of the predefined tiles to be requested, given as an integer. |
 | `height` | Optional | The height in pixels of the predefined tiles to be requested, given as an integer. If it is not specified in the JSON, then it defaults to the same as `width`, resulting in square tiles. |
 {: .api-table}
 
-Servers _SHOULD_ support requests for images with parameters specified by the `sizes` and `tiles` fields for all combinations of qualities and formats supported.
-
-The following shows a valid image information response, including the optional `sizes` and `tiles` properties.
+The `width` of a tile, or the combination of `width` and `height` if `height` is specified, _MUST_ be unique among the members of the `tiles` list.
```

Comment:
I suggest: "Objects in the `tiles` list _MUST_ each have a unique combination of `width` and `height`, where `height` = `width` if not explicitly specified."